biorbd_optim/constraints.py

```diff
@@ -216,3 +252,19 @@ def continuity_constraint(ocp):
             for i in range(ocp.nlp[0]["nx"]):
                 ocp.g_bounds.min.append(0)
                 ocp.g_bounds.max.append(0)
+
+
+class Correct:
+    @staticmethod
+    def parameters(name, elements, nb):
+        if not isinstance(elements, (list, tuple)):
+            elements = (elements,)
+        for element in elements:
+            if not isinstance(element, int):
+                raise RuntimeError(str(element) + " is not a valid index for " + name + ", it must be a " + str(type))
+            if element < 0 or element > nb:
+                raise RuntimeError(
+                    str(element) + " is not a valid index for " + name + ", it must be between 0 and " + str(nb - 1)
+                )
+
+    # TODO: same security in objective_function
```

Comment:
Could you find a better name? It seems that this is just a static private method...